maintenance/wikia/SUS-5726.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+require_once __DIR__ . '/../Maintenance.php';
+
+class SUS5726 extends Maintenance {
+
+	public function execute() {
+		global $wgUser;
+		$wgUser = User::newFromName( Wikia::BOT_USER );
+		foreach ( $this->allWikis() as $wiki ) $this->updateThemeSettings( $wiki->city_id );
+	}
+
+	private function allWikis() {
+		global $wgExternalSharedDB;
+		return wfGetDB( DB_SLAVE, [], $wgExternalSharedDB )
+			->select(
+				'city_list',
+				'city_id',
+				[ 'city_public = 1' ],
+				__METHOD__,
+				[ 'ORDER BY' => 'city_id ASC' ]
+			);
+	}
+
+	private function updateThemeSettings( string $wiki ) {
+		$var = WikiFactory::getVarValueByName( 'wgOasisThemeSettings', $wiki ) ;
```

Comment:
Just wondering: did you consider using `ThemeSettings` class instead of accessing the `wgOasisThemeSettings` directly? It has the advantage of preserving the history of the changes. Although I'm not sure you need it here.